db/reviewsDb.js

```diff
@@ -0,0 +1,81 @@
+var mongoose = require('mongoose');
+var faker = require('faker');
+var DateGenerator = require('random-date-generator');
+
+mongoose.connect('mongodb://localhost/reviews', { useNewUrlParser: true });
+
+var db = mongoose.connection;
+
+db.on('error', console.error.bind(console, 'CONNECTION ERROR: '));
+db.once('open', function () {
+  console.log("WE'RE CONNECTED")
+});
+
+var reviewSchema = new mongoose.Schema({
+  houseId: Number,
+  guestName: String,
+  date: Date,
+  guestReview: String,
+  guestImage: String,
+  hostName: String,
+  hostResponse: String,
+  hostImage: String,
+  ratings: {
+      location: Number,
+      checkIn: Number,
+      value: Number,
+      communication: Number,
+      accuracy: Number,
+      cleanliness: Number
+  }
+});
+
+var Review = mongoose.model('Review', reviewSchema);
+
+var data = [];
+
+var zeroThroughFive = function getRandomArbitrary(min, max) {
+  var num = Math.random() * (max - min) + min;
+  return num.toFixed(1);
+};
+
+var startDate = new Date(2018, 01, 01);
+var endDate = new Date(2019, 10, 10);
+//DateGenerator.getRandomDateInRange(startDate, endDate);
+
+var populator = function () {
+
+  for (var i = 1; i <= 10; i ++) {
+    var start = 0;
+    while (start <= 10) {
+      var each = {
+        houseId: i,
```

Comment:
Logic currently looks like there are 10 houseIds and per houseId # there are 10 reviews. A future consideration could be to randomize the houseId #  and randomize the # of comments to make the fake data even more random. Nice work! 👍